component/scripts/motd_gen.sh

```diff
@@ -3,7 +3,8 @@ set -exo pipefail
 
 jq 'del(.metadata) | .metadata.name = "motd" | .data.message = ([.data.message] + input | join("\n\n"))' \
     <(kubectl -n openshift get cm motd-template -ojson) \
-    <(kubectl get consolenotifications -l appuio.io/notification=true -ojson | jq '[.items[].spec.text]') \
+    <(kubectl get consolenotifications -l appuio.io/notification=true -ojson |
+      jq '[.items[]|.spec.text,if .spec.link.text then "\(.spec.link.text): \(.spec.link.href)" else "" end|select(length > 0)]') \
```

Comment:
it's safe to just check for `spec.link.text` (or href), since both fields are mandatory if `spec.link` is given.